app/src/main/java/com/example/surfstop/MainActivity.java

```diff
@@ -56,6 +60,9 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
     }
 
     public void onLogoutButton(MenuItem item) {
+        // Delete user cache
```

Comment:
This comment isn't really adding much value. Why you're deleting the cache is the value you can provide in the comment, i.e. "// Clear the cache to ensure any temporary photos the user took have been cleaned up when we log out" 

Otherwise this is roughly what we've got:
// dog
Dog dog;